api/internal/localizer/localizer.go

```diff
@@ -75,9 +76,65 @@ func (lc *Localizer) Localize() error {
 	return nil
 }
 
-// TODO(annasong): implement
-// processKust returns a copy of the kustomization at kt with paths localized.
-func (lc *Localizer) processKust(kt *target.KustTarget) *types.Kustomization {
+// processKust returns a copy of the kustomization at kt with the patches field localized
+func (lc *Localizer) processKust(kt *target.KustTarget) (*types.Kustomization, error) {
 	kust := kt.Kustomization()
-	return &kust
+	for name, patches := range map[string][]types.Patch{
+		"patches":         kust.Patches,
+		"patchesJson6902": kust.PatchesJson6902,
+	} {
+		for i := range patches {
+			if patches[i].Path != "" {
+				newPath, err := lc.localizeFile(patches[i].Path)
+				if err != nil {
+					return nil, errors.WrapPrefixf(err, "unable to localize %s path", name)
+				}
+				patches[i].Path = newPath
+			}
+		}
+	}
+	return &kust, nil
+}
+
+// localizeFile localizes file path and returns the localized path
+func (lc *Localizer) localizeFile(path string) (string, error) {
+	content, err := lc.ldr.Load(path)
+	if err != nil {
+		return "", errors.Wrap(err)
+	}
+
+	var locPath string
+	if loader.IsRemoteFile(path) {
+		// TODO(annasong): check if able to addLocalizeDir() once implemented
+		_ = lc.addLocalizeDir()
```

Comment:
Still need to call unimplemented function; otherwise, `linter` complains that unimplemented functions are not referenced.

Reply:
suggest deleting the `addLocalizeDir` completely and implemented in a follow up PR. 
```
if loader.IsRemoteFile(path) {
  locPath = locFilePath(path)
```